backend/app/resources.py

```diff
@@ -0,0 +1,87 @@
+import time
+from flask import make_response, request, Response, jsonify
+from app import app
+from .setup_db import client, users_collection, resources_collection, projects_collection
+from .auth import check_hash, encode, require_jwt, get_hash, User
+
+from urllib.parse import unquote
+
+
+@app.route('/resources', methods=['GET'])
+@require_jwt
+def all_resources():
+    return jsonify(resources_collection)
```

Comment:
Not sure if this will work. Try `collection.find()` and only collect the fields specified in OpenAPI.